build/repositories.bzl

```diff
@@ -130,7 +130,7 @@ def build_repositories():
     luarocks_repositories()
 
     kong_resty_websocket_repositories()
-    github_cli_repositories()
+    github_repositories()
```

Comment:
this is actually correct, since we are downloading the github CLI (`gh`), i would suggest to keep the original naming

Reply:
Yes, I want to change the name because we use this function to download other internal components in the EE version.
But now I think we can separate them into two functions in the EE version. I will do it manually during the cherry-pick